### PR TITLE
update one click install button

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: frontend/src/app/components/MainInstallButton.js [hunk 1]
```diff
@@ -11,14 +11,15 @@ export default class MainInstallButton extends React.Component {
   }
 
   install(e) {
-    const { requireRestart, sendToGA, eventCategory, hasAddon, installAddon, installCallback } = this.props;
+    const { requireRestart, sendToGA, eventCategory, eventLabel, hasAddon, installAddon, installCallback } = this.props;
```
Comment (johngruen) [inline frontend/src/app/components/MainInstallButton.js:14]: Big change here is that we're doing a custom label as well

File: frontend/src/app/components/MainInstallButton.js [hunk 2]
```diff
@@ -11,14 +11,15 @@ export default class MainInstallButton extends React.Component {
...
     if (installCallback) {
+      this.setState({ isInstalling: true });
```
Comment (johngruen) [inline frontend/src/app/components/MainInstallButton.js:16]: called here to add load state

File: frontend/src/app/components/MainInstallButton.js [hunk 3]
```diff
@@ -34,12 +35,23 @@ export default class MainInstallButton extends React.Component {
     );
   }
 
+/*eslint-disable */
```
Comment (johngruen) [inline frontend/src/app/components/MainInstallButton.js]: oops i should take this out

File: frontend/src/app/components/MainInstallButton.js [hunk 4]
```diff
@@ -34,12 +35,23 @@ export default class MainInstallButton extends React.Component {
...
+  renderOneClickInstallButton(title) {
```
Comment (johngruen) [inline frontend/src/app/components/MainInstallButton.js:38]: Abstracted this out to split the strings

File: frontend/src/app/containers/ExperimentPage.js [hunk 5]
```diff
@@ -651,12 +650,14 @@ export class ExperimentDetail extends React.Component {
 
     let installAddonPromise = null;
     if (!this.props.hasAddon) {
-      const { installAddon, requireRestart, eventCategory } = this.props;
-
+      const { installAddon, requireRestart } = this.props;
+      const eventCategory = 'ExperimentDetailsPage Interactions';
```
Comment (johngruen) [inline frontend/src/app/containers/ExperimentPage.js:654]: have to define these here b/c they aren't actually props

File: locales/en-US/app.ftl [hunk 6]
```diff
@@ -36,10 +36,14 @@ landingExperimentsTitle = Try out the latest experimental features
 # Related to the installation of the Test Pilot add-on.
 [[landingInstall]]
 landingInstallButton = Install the Test Pilot Add-on
-landingInstallButtonOneClick = Install Test Pilot and Enable {$experimentTitle}
 landingInstallingButton = Installing...
 landingInstalledButton = Choose your features
 
+# Related to a one click to install test pilot and an experiement.
```
Comment (fzzzy) [inline locales/en-US/app.ftl]: Typo for "experiment"

**Comment by fzzzy** [comment]
Looks great. r+ with one small typo changed

File: locales/en-US/app.ftl (hunk 6 above)
Comment (johngruen) [reply locales/en-US/app.ftl]: oops poops

**Pull request merged**